package.json

```diff
@@ -9,7 +9,7 @@
 	"license": "MIT",
 	"main": "index.js",
 	"browser": {
-		"./implementation": "./implementation.browser.js"
+		"./implementation.js": "./implementation.browser.js"
```

Comment:
Would doing break extensionless requires? Meaning, if someone requires `./implementation` or `./implementation.js` they should be getting the same thing.

It seems like a much better plan to wait for esbuild to fix it (or use another bundler) rather than trying to patch parts of the ecosystem.